lib/plugins/features/movies/controller.js

```diff
@@ -4,5 +4,27 @@ const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
   return new Movie().save(payload)
-  .then((movie) => new Movie({ id: movie.id }).fetch());
+    .then((movie) => new Movie({ id: movie.id }).fetch());
```

Comment:
nit: no indent of `.then`, i think this is a setting you can change in your text editor? looks like a bunch of them got indented